README.md

```diff
@@ -187,6 +187,8 @@ $ mbedls
 
 If you want to use ```mbedls``` in your toolchain, continuous integration or automation script and do not necessarily want to use the Python module ```mbed_lstools``` - this solution is for you.
 
+On some Linux systems, USB mass storage devices are not automatically mounted and do not show up when running `mbedls` by default. If you would like list these not mounted devices, you can run mbed-ls with the `-u` option: `$ mbedls -u`.
```

Comment:
I think you meant "If you would like to list" vs "If you would like list".

Reply:
You are correct.